src/base/solvers/buildCplexProblemFromCOBRAStruct.m

```diff
@@ -0,0 +1,63 @@
+function  cplexProblem = buildCplexProblemFromCOBRAStruct(Problem)
+% Build a cplex object from the given LP problem in COBRA Format
+% USAGE:
+%    cplexProblem = buildCplexProblemFromCOBRAStruct(LPproblem)
+%
+% INPUT:
+%    LPproblem:         A COBRA style Problem with the following fields:
+%                        * .A - The equality and in equality matrix
+%                        * .b - The right hand side values of the constraints
+%                        * .ub - the upper bounds of the variables
+%                        * .lb - the lower bounds of the variables
+%                        * .osense - The objective sense (-1 for max, 1 for min)
+%                        * .c - the objective coefficient vector for the linear part
+%                       OPTIONAL:
+%                        * .F - The objective coefficient matrix for the quadratic part.
+%                        * .varType - The variable types for mixed integer problems ('I', integer, 'C', continous,'B' binary)
+%                        * .b_L - left hand sides of the constraints (will only be used if csense is empty)
+%                        * .csense - The constraint senses, Default assumption is all 'E'
+%                        * .x0 - Basis to use 
+%
+
+
+try
```

Comment:
Yes, in theory this could be done. 
Its just, since we are creating the struct here anyways it fits quite well in here.

Reply:
But then we have a redundant check for the installation of CPLEX, which would be nice to have it only centrally in `changeCobraSolver`.

Reply:
I only used the code present in solveCobraLP, in order not to change behaviour.

Reply:
Actually, since solver is an allowed parameter in the optional parameters of `solveCobraXYZ`, this should stay in here, as otherwise the error thrown is uninformative.